recipe/meta.yaml

```diff
@@ -9,24 +13,26 @@ source:
   sha256: b2eb652d9221bdeb65772503891d8bfcfc36b3b1a2c9bb35b9d247a08965fd5d
 
 build:
-  number: 0
+  number: {{ build }}
+  string: {{ license_family }}_h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}
   skip: true  # [win]
 
 requirements:
   build:
-    - gnuconfig  # [not win]
+    - make
+    - gnuconfig
     - {{ compiler('c') }}
     - {{ stdlib("c") }}
     - {{ compiler('cxx') }}
     - pkg-config
   host:
-    - make  # [linux]
+    - make
     - bzip2
     - fftw
     - fontconfig
     - fonts-conda-forge
     - freetype
-    - ghostscript
+    - ghostscript  # [license_family=="agpl"]
```

Comment:
@isuruf  comment on matrix that this should be on build only. Sorry but I merged before I saw the comment.